ass2/index.html

```diff
@@ -0,0 +1,150 @@
+      <select>
+        <option>J</option>
+        <option>c</option>
+      </select>
+      <span>
+    </div>
+    <div class="clear"></div>
+    <ul class="members-list">
+    <li>A</li>
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
+        <div class="user-info">
+          <p class="user-name">A Stark</p>
+          <p class="user-email">astark@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
+        <div class="user-info">
+          <p class="user-name">Agtha Alev</p>
+          <p class="user-email">agtha@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+      <li> 
+        <div class="letter-style"><h3>A</h3></div>
+        <div class="user-info">
+          <p class="user-name">Alec Applelbuam</p>
+          <p class="user-email">alec@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+    </ul>
+    <ul class="members-list">
+    <li>J</li>
+      <li> 
+        <div class="letter-style"><h3>J</h3></div>
+        <div class="user-info">
+          <p class="user-name">John Doe (owner)</p>
+          <p class="user-email">jdoe@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+      <li> 
+        <div class="letter-style"><h3>J</h3></div>
+        <div class="user-info">
+          <p class="user-name">John Snow</p>
+          <p class="user-email">John@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+      <li> 
+        <div class="letter-style"><h3>J</h3></div>
+        <div class="user-info">
+          <p class="user-name">Jonnae Applelbuam</p>
+          <p class="user-email">jonnae@acme.com</p>
+        </div>
+        <button class="make-owner button">Make owner</button>
+      </li> 
+    </ul>
+    <button class="plus-button"><i class="icon-plus"></i></button>
+    <div class="clear"></div>
+    </section>
+
+    <section class="organization-info">
+    <h1>Organization Info </h1>
+    <h3>Acme Corp</h3>
+    <div class="organization-contact">
+      <span class="icon-map"></span>
+      <div># 206, Money Chambers <br>
+        K H Road, Shanti Nagar,<br>
+        Bangalore<br>
+        Karnataka <br>
+        India<br>
+        560027<br>
+      </div>
+      <span class="icon-phone"></span>
+      <div>+91 01234567890</div>
+      <span class="icon-sphere"></span>
+      <div>Asia/Kolkata</div>
+      <img src="img/map.png" class="map-image">
+      <button class="edit-button button">Edit
+    </div>
+    <div class="clear"></div>
+    <button class="delete button">Delete Organization</button>
+    <form>
+      <input type="text" class="form-info" placeholder="Transfer ownership">
+      <input type="text" class="form-info" placeholder="Enter Email Id">
```

Comment:
@priyanka0901 In placeholder you asking user to add email id but for input field you are passing input type as `text` instead of `email`